internal/clientconn/conn.go

```diff
@@ -247,7 +254,7 @@ func (c *conn) run(ctx context.Context) (err error) {
 	}
 }
 
-// Route routes the message.
+// route sends request to a handler's command based on the op code provided in the request header.
```

Comment:
I think we should change the next comment line `Route` -> `route` also.

Reply:
Well, the next like is the beginning of a new sentence, and I'm not sure if we should start it from a lowercased letter.

Reply:
Rephrased.